src/Bicep.Core.Samples/Files/AKS_LF/main.sourcemap.json

```diff
@@ -0,0 +1,286 @@
+{
```

Comment:
It's great that you've added sourcemap files to the set of baselines so that we'll be able to catch any formatting changes in future.

As a reviewer however, it's difficult to look at this file and say "yup, everything is working as expected". In addition to having the sourcemaps generated, I think it would also be a good idea to output a file which visually ties the bicep, sourcemap and JSON together, so that the reviewer can easily tell whether anything is off.

Take a look at https://github.com/Azure/bicep/blob/main/src/Bicep.Core.Samples/Files/InvalidExpressions_LF/main.diagnostics.bicep for an example - we generate these files by collecting diagnostics and annotating the original source file with them - making it easy for reviewers to catch whether a particular change introduces formatting changes, off-by-one errors, missing diagnostics, positioning errors etc.

Here's the test which generates this file: https://github.com/Azure/bicep/blob/6afa660cba723c79181b98b80f9965234e94857e/src/Bicep.Core.IntegrationTests/Semantics/SemanticModelTests.cs#L32-L55. We have various helper methods which may be useful. Feel free to ping me or anyone else on the team offline if you'd like more pointers or to discuss in any more depth.

Reply:
Thanks for detailed suggestion, will take a look!

Reply:
OK took a first stab at this--the tests' readability is definitely much improved. Let me know if you have ideas for further improvement.